src/register.js

```diff
@@ -5,8 +5,23 @@ import {
   getComponentNameFromFilename
 } from "./util";
 
-export default function registerStories(req, fileName, sbInstance, plugins) {
-  const { action, withKnobs, text, color, select, boolean } = plugins;
+export default function registerStories(req, fileName, sbInstance, plugins, decorators, storyOptions) {
```

Comment:
I'm not in love with this function signature having 6 arguments. I think we should convert this to a single hash and destructure those individual args.

Reply:
I agree, I've updated it to a single object that gets destructured but I'm thinking wether we should leave 
```req, filename, sbInstance``` as top level args. But put ```{plugins, decorators, storyOptions}``` under one object that we destructure. They feel logically grouped in my head, almost like an overall options or modifiers object. Not sure if you agree